hw/formal/README.md

```diff
@@ -324,6 +324,39 @@ which lists the total number of assertions and the number of proven, vacuous,
 covered and failing assertions for each block. CRASH identifies modules that
 fail to run VC Formal.
 
+## How To Run Connectivity Test on OpenTitan
+
+### Cadence JapserGold
+If you have access to JasperGold's connectivity app from Cadence, there are two
+options to run connectivity test:
+
+#### Batch Script
+You can trigger the `formal_conn.sh` batch script with the following command:
+```
+  cd hw/formal
+  formal_conn.sh -f {path_to_connectivity_csv_file}
+```
+This script also provides the following options:
+* `-top {top_name}`: specify which top_level to run. Default is `top_earlgrey`
+* `-p {core_path}`: specify which fusesoc core file to use. Default is
+  `lowrisc:systems:top_earlgrey`
+* `-batch`: run the connectivity script with batch mode
```

Comment:
I think batch mode should be the default. I will recommend providing `-gui` switch instead to override batch and use the interactive mode instead.

Reply:
Okay sure, I can update script and that :)